code/_core/datum/damagetype/ranged/laser/rifle.dm

```diff
@@ -44,7 +44,7 @@
 
 	//How much armor to penetrate. It basically removes the percentage of the armor using these values.
 	attack_damage_penetration = list(
-		LASER = AP_AXE*0.9,
-		HEAT = AP_AXE*0.1,
-		RAD = AP_AXE*0.1
-	)
\ No newline at end of file
+		LASER = AP_AXE*9999,
+		HEAT = AP_AXE*9999,
```

Comment:
Just for conformity, all large values should be written as INFINITY as this specific number doesn't run armor checks instead of running armor checks with large values.

Reply:
👍